packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme_desktop.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be found
+// in the LICENSE file.
+
+import 'ide_theme.dart';
+
+/// Load any IDE-supplied theming.
+IdeTheme getIdeTheme() => null;
```

Comment:
dart style: methods should not start with the word get.

Reply:
Any preference to renaming this (`parseIdeTheme`? `readIdeTheme`?) versus making it a getter?

My (minor) concern with making it a getter, is that it would be easy to accidentally use it globally from any code (rather than the reference passed down the tree from `main.dart`). With a method, that might be a little more obvious.

Right now it doesn't matter much (as long as the getter caches the result so it's not being re-read), but if we wanted to trigger rebuilds if it changed in future, having code reaching out to the global getter seems a bit weird.